packages/mui-material-next/src/Switch/Switch.tsx

```diff
@@ -7,12 +7,14 @@ import { refType, unstable_capitalize as capitalize } from '@mui/utils';
 import { unstable_composeClasses as composeClasses } from '@mui/base/composeClasses';
 import { alpha, darken, lighten } from '@mui/system';
 import SwitchBase from '@mui/material/internal/SwitchBase';
+import { OverridableComponent } from '@mui/types';
 import useThemeProps from '../styles/useThemeProps';
 import styled from '../styles/styled';
 import switchClasses, { getSwitchUtilityClass } from './switchClasses';
+import { SwitchOwnerState, SwitchProps, SwitchTypeMap } from './Switch.types';
 
-const useUtilityClasses = (ownerState) => {
-  const { classes, edge, size, color, checked, disabled } = ownerState;
+const useUtilityClasses = (ownerState: SwitchOwnerState) => {
+  const { classes, edge, size = 'medium', color = 'primary', checked, disabled } = ownerState;
```

Comment:
Aren't these default values already set in: https://github.com/mui/material-ui/blob/be63176220135368d09daaf4b5c6f92060267ab4/packages/mui-material-next/src/Switch/Switch.tsx#L225 ?

Reply:
@mj12albert Yes but type problem as described in #39935